### ISSUE TITLE
Notification not handled if app was killed

### ISSUE THREAD

**Issue by otusweb** [urbanairship/react-native-module#159]
# Please fill out this template when submitting an issue.
All lines beginning with an ℹ symbol indicate information that's important for you to provide to ensure your issue is resolved as quickly and efficiently as possible.

❗For how-to inquiries involving Urban Airship functionality or use cases, please
contact (support)[https://support.urbanairship.com/hc/en-us].

# Preliminary Info

### What Urban Airship dependencies are you using?
ℹ Please include Urban Airship dependencies and their versions here.
 
    "urbanairship-react-native": {
      "version": "2.0.2",
      "resolved": "https://registry.npmjs.org/urbanairship-react-native/-/urbanairship-react-native-2.0.2.tgz",
      "integrity": "sha512-TSr94Sv513Y1vXBFhC+7oUglg9ID3HoPs6PIUOioT9mPBbx/JZRVTrbvaT29dBjLPs2MyrV8VevCvsCUxcfRJA=="
    }

### What are the versions of any relevant development tools you are using?
ℹ If applicable, please include any relevant information about the version of your
IDE, plugin(s), or package manager(s) here.

# Report

### What unexpected behavior are you seeing?
ℹ Please provide a brief description of the unexpected behavior here.
Push notifications that include a deep link are not handled correctly if the app was killed before the notification is sent. The app is launched, but the deep link is not followed.
I'm using React native navigation also to handle my navigation needs.  I have the following code in my app.js but the callback is never called. 

```
UrbanAirship.addListener("deepLink", event => {
      const link = event.deepLink;
      console.log("receive event");
  xxxxx
    });
```

### What is the expected behavior?
ℹ Please provide a brief description of the expected behavior here.
I expect the deepLink call back to be called so the app can handle the deep link.

### What are the steps to reproduce the unexpected behavior?
ℹ Please provide reproduction steps here.
Create a react native app
add react native navigation
Configure Urban airship
add a deep event listener
run the app
send a notification with a deep link

### Do you have logging for the issue?
ℹ Please provide links to gists of any pertinent logs here.

**Comment by rlepinski** [comment]
What platform? If its iOS then you need to update to 2.0.3 or 2.1.1

**Comment by otusweb** [comment]
Yes, iOS, i'll try updating tomorrow. Thanks for the quick response.

**Comment by otusweb** [comment]
That fixed it thank you
